python/hail/expr/builders.py

```diff
@@ -283,3 +285,25 @@ def or_missing(self):
             raise ExpressionException("'or_missing' cannot be called without at least one 'when' call")
         from hail.expr.functions import null
         return self._finish(null(self._ret_type))
+
+    @typecheck_method(message=str)
```

Comment:
You need to add `or_error` to the description in the Warning message above for CaseBuilder.

Reply:
I think changing it to just "case" would be best.